docs/tips-n-tricks/Wormholes/gas-harvesting/ninja-huffing.md

```diff
@@ -18,15 +18,15 @@ Only one site cannot be ninja-huffed : the [Ordinary Perimeter Reservoir](https:
 
 ### Ships
 
-All ships and fits can ninja-huff
+All ships and fits can ninja-huff, as it requires no specific gear.
```

Comment:
I disagree with this. I want to leave the requirements section as simple as possible. I feel this extra bit adds nothing.